torchok/tasks/onnx.py

```diff
@@ -57,42 +59,53 @@ def __init__(self, hparams: DictConfig, path_to_onnx: str, providers, **kwargs):
         logging.info(f'ONNX model output names: {output_names}')
 
     def forward(self, x: Tensor) -> Tensor:
+        """Is not supported"""
         pass
 
     def forward_with_gt(self, batch: Dict[str, Any]) -> Dict[str, Tensor]:
+        """Is not supported"""
         pass
 
     def as_module(self) -> nn.Sequential:
+        """Is not supported"""
         pass
 
-    def foward_infer(self, inputs: Dict[str, Tensor]) -> Dict[str, Tensor]:
-        """Forward onnx model."""
+    def foward_infer(self, batch: Dict[str, Tensor]) -> Dict[str, Tensor]:
+        """Forward onnx model.
 
-        for input in self.inputs:
+        Args:
+            batch: Dictionary with the items specified in Dataset
+
+        Returns:
+            Dictionary with the output items that are specified in model onnx file.
+        """
+
+        batch_dim = 1
```

Comment:
Why is `batch_dim` hard coded? We expected this task to work with dynamic batches

Reply:
Before it was None, I can return this. It is not hardcoded, check two lines below